app/assets/stylesheets/application.scss

```diff
@@ -15,22 +15,266 @@
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kung-Jin {
+    background-image: image-url("character_backgrounds/Kung-Jin-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Kung-Lao {
+    background-image: image-url("character_backgrounds/Kung-Lao-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Liu-Kang {
+    background-image: image-url("character_backgrounds/Liu-Kang-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Mileena {
+    background-image: image-url("character_backgrounds/Mileena-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Quan-Chi {
+    background-image: image-url("character_backgrounds/Quan-Chi-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Raiden {
+    background-image: image-url("character_backgrounds/Raiden-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Reptile {
+    background-image: image-url("character_backgrounds/Reptile-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Scorpion {
+    background-image: image-url("character_backgrounds/Scorpion-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Shinnok {
+    background-image: image-url("character_backgrounds/Shinnok-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Sonya-Blade {
+    background-image: image-url("character_backgrounds/Sonya-Blade-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Subzero {
+    background-image: image-url("character_backgrounds/Subzero-Left.png");
+    background-attachment: fixed;
+    background-image: no-repeat;
+    background-size: cover;
+    background-color: black;
+    position: relative;
+  }
+
+  .Takeda {
+    background-image: image-url("character_backgrounds/Takeda-Left.png");
```

Comment:
Properties should be ordered background-attachment, background-color, background-image, background-image, background-size, position